internal/scraper/testdata/traceroute.txt

```diff
@@ -0,0 +1,29 @@
+# HELP probe_duration_seconds Returns how long the probe took to complete in seconds
```

Comment:
Q: are these updated? I don't see `probe_traceroute_total_hops` in this metrics list